assets/js/localization.js

```diff
@@ -60,14 +60,14 @@ $(document).ready(function () {
         var pathname = window.location.pathname;
 
         if(window.history.replaceState && !pathname.endsWith('/index.debug.html')) {
-            var urlParams = [], urlParamNames = ['dir', 'choice'];
+            var urlParams = [], urlParamNames = ['dir', 'choice', 'q1', 'q2', 'q3'];
```

Comment:
Is there any reason we can't use q for all of them? You're only in one mode (trans/gen/ana) at any one time.